Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM golang:1.13.0 AS build
+FROM golang:1.13 AS build
 WORKDIR /deck
-COPY go.mod ./
-COPY go.sum ./
+COPY . . 
```

Comment:
Please revert this change.
It is intentional that we only first copy over `go.mod` and `go.sum`. 

This gives Docker image caching, meaning that every time we execute a `docker build`, it won't result in a new hash and re-download all the modules in the next step.